src/java.base/aix/classes/sun/nio/fs/CloneFile.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package sun.nio.fs;
+
+import java.io.IOException;
+import sun.nio.ch.IOStatus;
+
+/**
+ * Implements a {@code clone) method for use by {@code UnixCopyFile} on AIX.
+ */
+final class CloneFile {
+    private CloneFile() { }
+
+    /**
+     * Clones the file whose path name is {@code src} to that whose path
+     * name is {@code dst} using a platform-specific system call.
+     *
+     * @implSpec
+     * The implementation in this class always returns
+     * {@code IOStatus.UNSUPPORTED}.
+     *
+     * @param src the path of the source file
+     * @param dst the path of the desintation file (clone)
```

Comment:
typo `desintation`. It seems it was copied in all CloneFile.java files.

Reply:
> typo `desintation`. It seems it was copied in all CloneFile.java files.

Thanks, I think I need a spell checker.

Reply:
Fixed in 9810352d8ba77fd0de56ac24540c3714400a455e.